app/views/search/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_tag :search_index, method: :get do %>
+  <p>Your EV Range (in miles)<%= select_tag :ev_range, options_for_select(ranges) %></p>
```

Comment:
I have routes for this - I would recommend changing this to a POST that routes to a `search#create`

Reply:
will make changes and push again.